sellotape/sellotape_dj/chat/views.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.decorators import login_required
+from django.utils.safestring import mark_safe
+from django.shortcuts import render
+import json
+
+def index(request):
```

Comment:
You better lint/reformat your code, 
Once you'll implement a CI with a linter, it'll fail in cases like this where there are no 2 lines between functions

Reply:
We'll add a reformat PR to address this format issues, it happens in other parts of the app aswell.